### ISSUE TITLE
Unable to find appropriate information and example of how to reference WCAG 2.1 in a regulation

### ISSUE THREAD

**Issue by maryjom** [w3c/wai-website#1181]
### Resource URL

https://www.w3.org/2013/02/stdref

### Description

I did a Google search trying to find a reference to tell me what is the correct format and content for someone to reference WCAG 2.1 (with the appropriate W3C information, copyright, etc.) So far I haven't been able to find anything useful.  The one reference I did find was incorrect and the reason I'm opening this issue.

Apologies if this is not the right repository (please transfer over to the right on, if needed). The issue I found is on the page [Normative References to W3C Standards](https://www.w3.org/2013/02/stdref)

At the bottom of this page is the following:

> ## Creating a Bibliography Entry for a W3C Recommendation
> W3C offers a [tool for generating a bibliography](https://www.w3.org/2002/01/tr-automation/tr-biblio-ui) of W3C Recommendations, Working Group Notes, and other Technical Reports.

However, the link doesn't take you to a tool to help you generate a bibliography, but instead a tool that helps you find a particular standard.

Additionally, I cannot find any useful examples of appropriate document references that should be used when referencing WCAG - a particular version and date, including from the [Document License](https://www.w3.org/copyright/document-license-2023/) page. This used to exist within the WCAG standard as an appendix section, but that has been removed.
